File: SDKs/amazonaws.com/ds/2015-04-16/java/lib/src/main/java/openapisdk/models/shared/DescribeEventTopicsResult.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * DescribeEventTopicsResult
 * The result of a DescribeEventTopic request.
**/
public class DescribeEventTopicsResult {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("EventTopics")
    public EventTopic[] eventTopics;
    public DescribeEventTopicsResult withEventTopics(EventTopic[] eventTopics) {
        this.eventTopics = eventTopics;
        return this;
    }
}

File: SDKs/amazonaws.com/ds/2015-04-16/python/src/sdk/models/shared/describeeventtopicsresult.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import eventtopic as shared_eventtopic


@dataclass_json
@dataclasses.dataclass
class DescribeEventTopicsResult:
    r"""DescribeEventTopicsResult
    The result of a DescribeEventTopic request.
    """
    
    event_topics: Optional[list[shared_eventtopic.EventTopic]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('EventTopics') }})
    
